### ISSUE TITLE
Question more than issue

### ISSUE THREAD

**Issue by blarg3891** [drtomasso/check-unifi-video#2]
I'm assuming that your script was meant to alert users when recording stopped on cameras or was broken and such.  What I have found is that many times the following error will appear in the error log file 1576797990.883 2019-12-19 23:26:30.883/UTC: ERROR  [uv.recording.info] Unable to close recording: Unable to update recording: 5dfc0718c2dced9cf01cb028 in CPSTask-1.  This prevents recording from occurring.  However, the Camera dashboard still shows last recording as "a few seconds ago....2 minutes ago" etc.  Have you had any experience with this as related to your script?  I need to play a bit more with this and verify but seems like we might not be notified based on your script in scenarios like this and will have to parse the logs for these types of errors.  Thoughts?

**Comment by foamrider** [comment]
No, haven't experienced those errors. That is a bug in Unifi Video software, so rather than changing the script to check for such errors, the Ubiquiti developers should fix their side of the code.

Adding a check if a video-file is actually present and valid, or parsing log-files would require a local script I guess. This script uses the API, so no local script on the NVR is needed. But it is possible to check the database for video-files and verifying that these exists, or parsing the log-files for certain errors.

**Comment by blarg3891** [comment]
Yeah, its true.  I did some more digging.  It looks like this error occurred after some patching and kernel updates.  I didnt notice because i saw that recordings were all saying "a few seconds ago" in the dash but this isnt really true had I looked at the logs.  I set up automatic emails for the unattended updates on Ubuntu, or I guess i could just turn auto updates off.  This way i'll just check the logs any time an update is done for now.  Maybe i'll build a script to check for errors in the recording log at some point...too busy at the moment.  I use Icinga and i think they have some log parsers out there already to use with the agent.  I guess I could setup something like graylog also..  I really wish Ubiquiti devs would have some better more reliable notification or an API like you said for checking for these types of problems.  I still really havent found a better NVR solution yet...so just gonna be more careful.  I mostly wanted to let you know about this issue in case you were relying on your script to always provide accurate information about recording status.  Just be careful because it may fool you :)
